controller: new release (#56)
NethServer/nethsecurity-docs: controller.rst

Controller
==========

The NethSecurity controller is an application that can run on `NethServer 8 (NS8) <https://ns.nethserver.org>`_.

> The controller enables to manage multiple NethSecurity installations remotely, called units.

Reviewer: ```suggestion
The controller allows remote control of multiple NethSecurity installations, called units.
```

The firewall can operate independently without the need for the controller.
The controller is an optional component that provides additional management and monitoring capabilities for the firewall.

The controller works by creating a secure connection between the central server and the units.
Each firewall registers with the server using a client called ns-plug.
Once registered, the server generates a VPN configuration that is sent back to the firewall

> This allows for secure communication between the controller and the unit.

Reviewer: ```suggestion
The VPN enables secure communication between the controller and the unit.
```

===============================

The controller can be installed on a NethServer 8 system from the Software Center. The module is named `NethSecurity Controller`.

After the installation, the controller must be configured. The configuration can be done using the NethServer 8 web interface.
The following parameters need to be set:

- `Controller hostname`: The fully qualified domain name for the controller, like: ``mycontroller.nethsecurity.org``.
- `Let's Encrypt certificate`: Enable or disable Let's Encrypt certificate for the controller web interface.
- `VPN details`: The OpenVPN network and netmask. When choosing the network, make sure it does not overlap with the existing networks inside all
  the units that will be connected to the controller.
- `VPN common name`: The OpenVPN certificate name which is also the name of the controller.
- `Administrator user and password`: The default controller admin user and password. You should change the password after the first login both
  for the controller and the Grafana interface.
- `Log retention`: The log retention period in days, default is 180 days.
- `Metrics retention`: The metrics retention period in days, default 15 days.

> After the configuration is done, the controller is ready to be used and can be accessed using a web browser at the configured hostname, like ``https://mycontroller.nethsecurity.org``.

Reviewer: ```suggestion
After completing the configuration, the controller is ready to be used and can be accessed using a web browser at the configured hostname, like ``https://mycontroller.nethsecurity.org``.
```

Users
=====

The controller has two types of users:

- **Administrator**: The administrator user is the only one that can create and manage users inside the controller.
- **User**: The user can manage the units and the firewall configurations.

The administrator user is created during the controller configuration. 

> The administrator user is the only one who can create and manage users within the controller.

Reviewer: Repeated from above: remove?

Author: Removed

> The administrator user is the only one who can create and manage users within the controller.

Author: ```suggestion
```
